pkg/ttnpb/lorawan.go

```diff
@@ -633,8 +634,9 @@ var errParsingSemanticVersion = unexpectedValue(
 
 // Validate reports whether v represents a valid MACVersion.
 func (v MACVersion) Validate() error {
+	max := MACVersion(len(MACVersion_name) - 1)
 	if v < 1 || v >= MACVersion(len(MACVersion_name)) {
-		return errExpectedBetween("MACVersion", 1, len(MACVersion_name)-1)(v)
+		return errExpectedBetween("MACVersion", fmt.Sprintf("%d(%s)", 1, MACVersion(1)), fmt.Sprintf("%d(%s)", max, MACVersion(max)))(v)
```

Comment:
- Keep `max` local or test `v > max`
- This formatting is not super beautiful; `1(MAC_V1_0)` or something?

Reply:
Meh